File: KL_LayangLayang.cpp
#include <iostream>
using namespace std;

int main()
{
    int menu;
    float x, y, d1, d2, keliling, luas;

    cout << "========= KELILING & LUAS LAYANG-LAYANG =========\n\n";
    cout << "1. Keliling Layang-Layang\n2. Luas Layang-Layang\n\n";
    cout << "Inputkan Nomor Menu        = "; cin >> menu;
    cout << endl;

    switch(menu){
        case 1 :
            cout << "Inputkan Sisi Pertama Layang-Layang = "; cin >> x;
            cout << "Inputkan Sisi Kedua Layang-Layang   = "; cin >> y;
            keliling = 2 * (x + y);
            cout << "\nKeliling Layang-Layang              = " << keliling << endl;
            break;
        case 2 :
            cout << "Inputkan Diagonal Pertama Layang-Layang    = "; cin >> d1;
            cout << "Inputkan Diagonal Kedua Layang-Layang      = "; cin >> d2;
            luas = 0.5 * d1 * d2;
            cout << "\nLuas Layang-Layang                         = " << luas << endl;
            break;
        default:
            cout << "Input Salah" << endl;
    }
}

File: KL_LayangLayang.java

import java.util.Scanner;

/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 *
 * @author USER
 */
public class KL_LayangLayang {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int menu;
        double x, y, d1, d2, keliling, luas;
        
        System.out.println("========= KELILING & LUAS LAYANG-LAYANG =========\n\n");
        System.out.println("1. Keliling Layang-Layang\n2. Luas Layang-Layang\n\n");
        System.out.print("Inputkan Nomor Menu        = ");
        menu = sc.nextInt();
        System.out.println("");
        
        switch(menu){
            case 1:
                System.out.print("Inputkan Sisi Pertama Layang-Layang = ");
                x = sc.nextDouble();
                System.out.print("Inputkan Sisi Kedua Layang-Layang   = ");
                y = sc.nextDouble();
                keliling = 2 * (x + y);
                System.out.println("\nKeliling Layang-Layang              = " +keliling);
                break;
            case 2:
                System.out.print("Inputkan Diagonal Pertama Layang-Layang    = ");
                d1 = sc.nextDouble();
                System.out.print("Inputkan Diagonal Kedua Layang-Layang      = ");
                d2 = sc.nextDouble();
                luas = 0.5 * d1 * d2;
                System.out.println("\nLuas Layang-Layang                         = " +luas);
                break;
            default:
                System.out.println("Input Salah");
        }
    }
}
